integration/deploy/manifest_test.go

```diff
@@ -348,8 +349,8 @@ func TestDeployOktetoManifestExportCache(t *testing.T) {
 
 // TestDeployRemoteOktetoManifest tests the following scenario:
 // - Deploying a okteto manifest in remote with a build locally
+// - Check that is not running on depot
 func TestDeployRemoteOktetoManifest(t *testing.T) {
-	t.Parallel()
```

Comment:
We need to remove the parallel because the test need to set fake env vars and not affect the rest of the tests